### ISSUE TITLE
[Feature Request] Default Off/Whitelist vs Blacklist behaviour

### ISSUE THREAD

**Issue by graeme-gemin** [m-khvoinitsky/dark-background-light-text-extension#316]
My suggestion is that as opposed to the current enabled/disabled toggle with blacklisting behaviour (i.e. the "configured pages" list allows you to change away from the default behaviour), there be a whitelist option that disables the extension by default, and have a list of sites on which the extension is enabled.

The use case for this is that while the extension is very useful, it is still less configurable than many site's built in appearance settings (by the nature of it's operating mechanism) which many people have configured on sites they frequent, but when using this extension those settings are overridden in favour of the extension options. Also I would rather see a new website as intended on first viewing in case the extension breaks it, as opposed to having to disable it for every site I visit.

Thanks

**Comment by pAX-dk** [comment]
+1
I totally agree. Just wanted to create a ticket about it.

**Comment by veita** [comment]
I agree. In >80% of all cases I find myself disabling the extension. Opting in would be much more convenient.
